signalfx/resource_signalfx_dashboard_group.go

```diff
@@ -37,6 +37,11 @@ func dashboardGroupResource() *schema.Resource {
 				Description: "Dashboard IDs that are members of this dashboard group. Also handles 'mirrored' dashboards.",
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
+						"config_id": &schema.Schema{
```

Comment:
I think we should also add a corresponding entry to the resource docs here: https://github.com/splunk-terraform/terraform-provider-signalfx/blob/main/website/docs/r/dashboard_group.html.markdown#argument-reference since we're adding a new resource field.